eureka-core/src/main/java/com/netflix/eureka/PeerAwareInstanceRegistry.java

```diff
@@ -674,14 +674,14 @@ public long getNumOfReplicationsInLastMin() {
     }
 
     /**
-     * Checks if the number of renewals is lesser than threshold.
+     * Checks if the number of renewals is lesser than threshold. This method is for servo metrics only.
      *
      * @return 0 if the renewals are greater than threshold, 1 otherwise.
      */
     @com.netflix.servo.annotations.Monitor(name = "isBelowRenewThreshold", description = "0 = false, 1 = true",
             type = com.netflix.servo.annotations.DataSourceType.GAUGE)
     public int isBelowRenewThresold() {
-        if ((getNumOfRenewsInLastMin() < numberOfRenewsPerMinThreshold)
+        if ((getNumOfRenewsInLastMin() <= numberOfRenewsPerMinThreshold)
```

Comment:
see issue #533. This is to align servo reporting logic with actual selfPreservation logic.